docs/source/JAX_for_LLM_pretraining.ipynb

```diff
@@ -453,14 +485,16 @@
     "                )\n",
     "        # Create a list of `TransformerBlock` instances.\n",
     "        # Each block processes input sequences using attention and feed-forward networks.\n",
-    "        self.transformer_blocks = [TransformerBlock(\n",
+    "        self.transformer_blocks = nnx.List([TransformerBlock(\n",
```

Comment:
nnx.List is a Flax 0.12 feature I believe.

Current AI Stack uses 0.11

Reply:
It's been a while since we've released. I can do a new release if that would help?